test/polyrith.lean

```diff
@@ -126,25 +126,25 @@ by test_polyrith "{\"data\":[\"(poly.const 1/1)\"],\"success\":true}" ["ff", "in
 
 example (x y : ℚ) (h1 : x*y + 2*x = 1) (h2 : x = y) :
   x*y = -2*y + 1 :=
-by test_polyrith "{\"data\":[\"(poly.const 1/1)\",\"(poly.const -2/1)\"],\"success\":true}" ["ff", "rat", "2", "[(((var0 * var1) + (2 * var0)) - 1), (var0 - var1)]", "((var0 * var1) - (((-1 * 2) * var1) + 1))"]  "linear_combination h1 - 2 * h2"
+by test_polyrith  "{\"data\":[\"(poly.const 1/1)\",\"(poly.const -2/1)\"],\"success\":true}" ["ff", "rat", "2", "[(((var0 * var1) + (2 * var0)) - 1), (var0 - var1)]", "((var0 * var1) - ((-2 * var1) + 1))"]  "linear_combination h1 - 2 * h2"
```

Comment:
Where's this extra space after `test_polyrith` coming from? I think I removed it in a previous commit.

Reply:
The `test_polyrith` lines are all generated by `create_polyrith_test`, I guess this is a bug there. I'll fix it.